lambda/src/main/scala/pricemigrationengine/model/AmendmentData.scala

```diff
@@ -162,7 +162,6 @@ object AmendmentData {
         .toRight(AmendmentDataFailure(s"Failed to get RatePlan for charges: $ratePlanCharges"))
 
       isZoneABC = zoneABCPlanNames contains ratePlan.productName
-      // check the Zone ABC rateplan is the currently active one -> effectiveEndDate property on the subscription, are we filtering out inactive Zone ABC rateplans in the bigquery query?? does the engine filter out inactive rateplans??
```

Comment:
this `ratePlan` variable is the currently active one. There's a function above which matches rate plans on the subscription and the invoice which extracts the active plan.